charts/dapr/crds/subscription.yaml

```diff
@@ -4,6 +4,18 @@ metadata:
   name: subscriptions.dapr.io
 spec:
   group: dapr.io
+  conversion:
+    strategy: Webhook
+    webhook:
+      clientConfig:
+        service:
+          namespace: replaceme # Patched by post-install webhook
+          name: dapr-webhook
+          path: /convert
+        #caBundle: Patched by post-install webhook
+      conversionReviewVersions:
+      - v1
```

Comment:
Do we have v1 for this CRD?

Reply:
No, but this requires either `v1` or `v1beta1` in the list of values or else it is invalid. :(